pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```

Comment:
Is it "# Fixes" or "Fixes #" ?

Reply:
I suppose to automatically link the PR to issue is "keyword" + "#issueid"?